includes/specials/SpecialContributions.php

```diff
@@ -552,14 +552,11 @@ function getQueryInfo() {
 
 		# Don't include orphaned revisions
 		$join_cond['page'] = Revision::pageJoinCond();
-		# Get the current user name for accounts
-		$join_cond['user'] = Revision::userJoinCond();
 
 		$queryInfo = array(
 			'tables'     => $tables,
 			'fields'     => array_merge(
 				Revision::selectFields(),
-				Revision::selectUserFields(),
```

Comment:
Is this one still needed anywhere?

Reply:
It's still used in few other places, let's remove after those tickets are handled.